uber/models/panels.py

```diff
@@ -114,6 +114,23 @@ def matched_attendees(self):
     def unmatched_applicants(self):
         return [a for a in self.applicants if not a.attendee_id]
 
+    @property
+    def guidebook_name(self):
```

Comment:
Finally, the breakdown of our Panels Guidebook export:
We... return all panel applications, and that oversight is one of the reasons I'm writing this out. I'll need to change this to only accepted panel apps.

Name: Panel's name.
Subtitle: The word "Panel" -- wasn't sure what else to put here.
Description: We take the description and add each applicant's "Other Credentials" after it, each preceded by two HTML linebreaks. Guidebook's docs indicate that this will work when importing directly into Guidebook.
Location: If the panel is associated with an event, we return that event's room's name. Otherwise, nothing.

Like the Marketplace, panelists don't submit images so we can't include them.